cmd/run.go

```diff
@@ -168,7 +168,9 @@ for existing bundle files or directories following the bundle structure.
 
 To skip bundle verification, use the --skip-verify flag.
 `,
+
 		Run: func(cmd *cobra.Command, args []string) {
+			cmdParams.rt.AddrChanged = cmd.Flags().Changed("addr")
```

Comment:
Nit: Can we move this inside `initRuntime`?

Reply:
Hey! :wave: 

I did try this before but I was having trouble figuring out how to access `cmd.Flags().Changed("addr")` inside the `initRuntime`

Maybe I am missing something obvious, could I get your advise on this? :pray:

Reply:
Why hasn't this worked before? You can pass the value of `cmd.Flags().Changed("addr")` in `initRuntime` maybe?

Reply:
Think I have solved this part, just doing some more testing :+1:

Reply:
Hi @ashutosh-narkar, based on what I have tried I have the following possible solutions but maybe I am overcomplicating it so would love your expert advice on this :muscle: 

1. Add the `*cobra.Command` parameter to `runCmdParams` and `initRuntime` function: we would be adding the *cobra.Command parameter to the `runCmdParams` struct, passing it to the initRuntime function, and updating the value of `AddrSetByUser` inside `initRuntime`. This approach requires modifying the function signature of initRuntime and updating all the tests accordingly.

2. Pass `cmd.Flags().Changed("addr")` value `in initRuntime`: Instead of passing the whole *cobra.Command parameter, we can pass only the value of `cmd.Flags().Changed("addr")` as an argument to `initRuntime`. This also requires modifying the function signature of `initRuntime` and updating all the tests accordingly.

3. Use a function to set `AddrSetByUser` before calling `initRuntime`: Create a new function to set the value of `AddrSetByUser` based on `cmd.Flags().Changed("addr")`. Call this function before invoking `initRuntime` in the Run function. This approach doesn't require modifying the initRuntime function signature, and the tests can remain unchanged.

What do you think of these? Or lastly, can leave it as is? :sweat_smile:

Reply:
I was talking about approach 2. There are just 4 tests that need to be updated so that's fine I think.

Reply:
Hey! think this is okay now... will resolve the conversation as I've pushed new changes. :+1: